.gitignore

```diff
@@ -45,3 +45,6 @@ compiler/.gradle/*
 # protobuf
 dubbo-serialization/dubbo-serialization-protobuf/build/*
 dubbo-demo/dubbo-demo-triple/build/*
+
+# global registry center
+test/zookeeper/
```

Comment:
git ignore pattern needs to be updated to `.tmp`

Reply:
I've fixed it.